tools/clang/test/CodeGenSPIRV/gs.emit.hlsl

```diff
@@ -0,0 +1,57 @@
+// Run: %dxc -T gs_6_0 -E main
+
+struct GsInnerOut {
+    float2 bar  : BAR;
+};
+
+struct GsPerVertexOut {
+    float4 pos  : SV_Position;
+    float3 foo  : FOO;
+    GsInnerOut s;
+};
+
+// CHECK: [[null:%\d+]] = OpConstantNull %GsPerVertexOut
+
+[maxvertexcount(2)]
+void main(in    line float2 foo[2] : FOO,
+          in    line float4 pos[2] : SV_Position,
+          inout      LineStream<GsPerVertexOut> outData)
+{
+// CHECK:            %src_main = OpFunction %void None
+// CHECK:            %bb_entry = OpLabel
+
+// CHECK-NEXT:         %vertex = OpVariable %_ptr_Function_GsPerVertexOut Function
+    GsPerVertexOut vertex;
+// CHECK-NEXT:                   OpStore %vertex [[null]]
+    vertex = (GsPerVertexOut)0;
+
+// Write back to stage output variables
```

Comment:
Please add a "CHECK" so that it becomes clear that these writes are occurring in the inner main function, and not in the wrapper function.